src/sentry/static/sentry/app/views/eventsV2/querycard.tsx

```diff
@@ -56,7 +61,7 @@ class QueryCard extends React.PureComponent<Props> {
   }
 }
 
-const StyledQueryCard = styled('button')`
+const StyledQueryCard = styled(Link)`
```

Comment:
Will this trigger DOM nesting warnings when the context menu is opened?

Reply:
Nope, the parent elements are `div` all the way up. I've checked in Chrome and FF.